pyemu/utils/geostats.py

```diff
@@ -872,6 +873,8 @@ def calc_factors_grid(
         var_filename=None,
         forgive=False,
         num_threads=1,
+        try_use_ppu=False,
```

Comment:
let's aim to phase this the default to True, over time.
Any danger of some docstring additions though?

Reply:
Good points.  I was thinking of a "soft launch" here to ease into it but maybe a docstring would be less ambiguous